p2p/stdnetwork/network.go

```diff
@@ -20,21 +20,22 @@ var (
 type network struct {
 	addrBookMap typesP2P.AddrBookMap
 
-	logger modules.Logger
+	logger *modules.Logger
 }
 
 func NewNetwork(bus modules.Bus, addrBookProvider providers.AddrBookProvider, currentHeightProvider providers.CurrentHeightProvider) (n typesP2P.Network) {
+	logger := logger.Global.CreateLoggerForModule("network")
```

Comment:
Rename local var to `log` or `networkLogger` to avoid overshadowing the import.

Ditto elsewhere.

![Screenshot 2023-02-09 at 10 29 29 AM](https://user-images.githubusercontent.com/1892194/217904937-86ce9413-319a-4950-8b80-42a99b52aca6.png)